scipy/optimize/tnc.py

```diff
@@ -90,7 +90,7 @@
 
 def fmin_tnc(func, x0, fprime=None, args=(), approx_grad=0,
              bounds=None, epsilon=1e-8, scale=None, offset=None,
-             messages=MSG_ALL, maxCGit=-1, maxfun=None, eta=-1,
+             messages=MSG_ALL, maxCGit=-1, maxfun=None, maxiter=None, eta=-1,
```

Comment:
I don't think that we need to add `maxiter` to the keyword arguments.

Reply:
I removed it.